content/04.results.md

```diff
@@ -10,4 +10,18 @@ place the grid of aurocs here for discriminator model
 place the results of random label function experiment
 
 ## Reconstructing Hetionet
-place figure of number of new edges that can be added to hetionet as well as edges we can reconstruct using this method
+After the discriminiative model has been calibrated, we estimated the number of edges that can be added/recalled from Hetionet v1.
+Figure {@fig:hetionet_reconstruction}, depicts the number of edges based on precision level.
+For Disease associates Gene (DaG) we can add 74 new edges while matintaining 70% precision.
+Likewise with Compound treats Disease (CtD) we can add 10 new edges while maintaining 70% precision.
+As for Compound binds Gene (CbG) and Gene interacts Gene (GiG) the models fail to reach precision higher than 0.5 and 0.6 respectively.
+Despite the low performance, we are still able to add new edges (7 for CbG at 47% precision and 9 for GiG at 56% precision) to Hetionet v1.
```

Comment:
How is precision determined here? Since you've done the probability calibration, could you just use the predicted probability as the precision of that point? Also note that 70% precision could actually be quite high if Hetionet is incomplete. In these cases, all your top predictions could be true, but yet you may still get lower precision since the truth is not completely in Hetionet.

Reply:
Precision is calculated using already established edges as positive while never before seen edges are treated as negative. You raise a good point as there are some edges that might be true even though they haven't been seen in Hetionet.

Reply:
Is this performance at recovering het.io on the test set?

Reply:
Performance here is using all edges in het.io as the evaluation set. After each sentence has been scored, I combined them together based on mention pair and take the max score for each pair.

Reply:
Since het.io is used for distant supervision, is there an issue with this + evaluation?

Reply:
Hmmm..  Based on past experiences, my intuition says there is an overfitting problem or at least there is a confounding issue with the evaluation. If I'm on the right path, I should have "held out" edges from distant supervision label functions and see if my method can predict the "held out" edges downstream.

Reply:
What if you evaluate this on only edges from the test set predictions?